trunk/SUAVE/Analyses/Aerodynamics/AVL_Inviscid.py

```diff
@@ -40,47 +39,27 @@
 # ----------------------------------------------------------------------
 #  Class
 # ----------------------------------------------------------------------
-## @ingroup Analyses-Aerodynamics
-class AVL_Inviscid(Aerodynamics):
-    """This builds a surrogate and computes lift using AVL.
 
-    Assumptions:
-    None
+class AVL_Inviscid(Aerodynamics):
+    """ SUAVE.Analyses.Aerodynamics.AVL
+        aerodynamic model that performs a vortex lattice analysis using AVL
+        (Athena Vortex Lattice, by Mark Drela of MIT).
+    """
```

Comment:
Why were the docstrings removed from this file?

Reply:
I think when I oped a new AVL branch I copied some old text over. Issue Resolved